src/qibojit/backends/clifford_operations_cpu.py

```diff
@@ -3,12 +3,22 @@
 import numpy as np
 from numba import njit, prange, uint64
 
+PARALLEL = False
 
-@njit("b1[:,:](b1[:,:], u8, u8)", parallel=True, cache=True)
+
+@njit("Tuple((u1[:], u1[:,:], u1[:,:]))(u1[:,:], u8)", parallel=PARALLEL, cache=True)
+def _get_rxz(symplectic_matrix, nqubits):
+    return (
+        symplectic_matrix[:, -1],
+        symplectic_matrix[:, :nqubits],
+        symplectic_matrix[:, nqubits:-1],
```

Comment:
Yes, the -1 was there since in the original version we did not need the last row of the symplectic matrix. However, now that the matrix is packed in chunks of 8 bits, the -1 does not correspond to the last row but actually to the last chunk, which could be by chance the last row plus some zeros for padding, but it can also be the last 8 rows of the original matrix. Long story short, we cannot isolate the last line anymore.

Reply:
Perfect, thanks for the explanation!